mix.exs

```diff
@@ -70,7 +70,9 @@ defmodule ElixirVictoria.MixProject do
     [
       "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
       "ecto.reset": ["ecto.drop", "ecto.setup"],
-      test: ["ecto.create --quiet", "ecto.migrate", "test"]
+      test: ["ecto.create --quiet", "ecto.migrate", "test"],
+      "cypress.open": ["cmd ./cypress-open.sh"],
+      "cypress.run": ["cmd ./cypress-run.sh"]
```

Comment:
This is a place where we can do aliases. If you want to run Cypress on its own you can do:

`mix cypress.run` for command line only, or
`mix cypress.open` to open up the GUI and actually watch it drive the browser. This is awesome for getting more detail around why acceptance tests are failing.